go.mod

```diff
@@ -1,80 +1,86 @@
 module github.com/hyperledger-labs/yui-relayer
 
-go 1.20
+go 1.21
 
 require (
-	cosmossdk.io/errors v1.0.0-beta.7
+	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/math v1.2.0
+	cosmossdk.io/x/evidence v0.1.0
+	cosmossdk.io/x/upgrade v0.1.0
 	github.com/avast/retry-go v3.0.0+incompatible
-	github.com/cockroachdb/errors v1.9.1
-	github.com/cometbft/cometbft v0.37.2
-	github.com/cometbft/cometbft-db v0.8.0
-	github.com/cosmos/cosmos-sdk v0.47.3
+	github.com/cockroachdb/errors v1.11.1
+	github.com/cometbft/cometbft v0.38.2
+	github.com/cometbft/cometbft-db v0.9.1
+	github.com/cosmos/cosmos-sdk v0.50.1
 	github.com/cosmos/go-bip39 v1.0.0
-	github.com/cosmos/gogoproto v1.4.10
-	github.com/cosmos/ibc-go/v7 v7.2.0
-	github.com/datachainlab/ibc-mock-client v0.3.3
-	github.com/prometheus/client_golang v1.15.1
+	github.com/cosmos/gogoproto v1.4.11
+	github.com/cosmos/ibc-go/modules/capability v1.0.0
+	github.com/cosmos/ibc-go/v8 v8.0.0
+	github.com/datachainlab/ibc-mock-client v0.3.4
+	github.com/prometheus/client_golang v1.17.0
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/viper v1.16.0
 	go.opentelemetry.io/otel v1.16.0
 	go.opentelemetry.io/otel/exporters/prometheus v0.39.0
 	go.opentelemetry.io/otel/metric v1.16.0
 	go.opentelemetry.io/otel/sdk/metric v0.39.0
-	golang.org/x/exp v0.0.0-20230515195305-f3d0a9c9a5cc
-	golang.org/x/sync v0.1.0
+	golang.org/x/exp v0.0.0-20231006140011-7918f672742d
```

Comment:
Can we switch from `golang.org/x/exp/slog` to `log/slog` in the standard library?

Reply:
Change `golang.org/x/exp/slog` to `log/slog`
https://github.com/hyperledger-labs/yui-relayer/pull/134/commits/fdaee6cb0c631ab36c7e0af7d26d3615ff849804